ocs_ci/ocs/bucket_utils.py

```diff
@@ -2054,12 +2071,33 @@ def create_aws_bs_using_cli(
     )
 
 
-def expire_objects_in_bucket(bucket_name):
+def upload_bulk_buckets(s3_obj, buckets, amount=1, object_key="obj-key-0", prefix=None):
+    """
+    Upload given amount of objects with sequential keys to multiple buckets
+
+    Args:
+        s3_obj: obc/mcg object
+        buckets (list): list of bucket names to upload to
+        amount (int, optional): number of objects to upload per bucket
+        object_key (str, optional): base object key
+        prefix (str, optional): prefix for the upload path
+
+    """
+    for bucket in buckets:
+        for index in range(amount):
+            s3_put_object(
+                s3_obj, bucket.name, f"{prefix}/{object_key}-{index}", object_key
+            )
+
+
+def expire_objects_in_bucket(bucket_name, new_expire_interval=None):
```

Comment:
I may be missing something here, but what's the use of the `new_expire_interval` argument in this util function?

If the purpose is to change the interval of the background worker that checks for expirations on noobaa-core, then using this fixture on the test itself should be enough IMO: https://github.com/red-hat-storage/ocs-ci/blob/27a124581690d1adc6080faa2b080bb1f360ac27/tests/conftest.py#L7129-L7148